tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/portal/PortalRuntimeComposite.java

```diff
@@ -102,7 +123,50 @@ public void widgetSelected( SelectionEvent e )
             }
         });
 
-        this.typeField = createReadOnlyTextField( "Detected bundle type" );
+        this.typeField = createReadOnlyTextField( Msgs.detectedBundleType );
```

Comment:
Lets make the background of the field dark, not make it look like it is editable.